ios/Extensions.swift

```diff
@@ -0,0 +1,37 @@
+import Foundation
+import SwiftUI
+import UIKit
+
+/**
+ Helper used to render UIView inside of SwiftUI.
+ */
+struct RepresentableView: UIViewRepresentable {
+  var view: UIView
+  
+  func makeUIView(context: Context) -> UIView {
+    let wrapper = UIView()
+    wrapper.addSubview(view)
+    return wrapper
```

Comment:
Let's add comment why it is necessary

https://github.com/callstack/react-native-pager-view/issues/819#issuecomment-2188285874

https://github.com/callstack/react-native-pager-view/pull/915